src/iexamine.cpp

```diff
@@ -855,6 +855,26 @@ void iexamine::toilet( player &p, const tripoint &examp )
     }
 }
 
+/** Toggle the lights in a overmap terrain*/
+void iexamine::toggle_lights( player &/*p*/, const tripoint &examp )
+{
+    map &here = get_map();
+    std:: string flag = "L_OFF";
+    std::string message = g->m.furn( examp ).obj().message;
+    std::string prompt = g->m.furn( examp ).obj().prompt;
+
+    // Checks to turn on or off
+    if( !here.has_flag_furn( flag, examp ) ) {
+        flag = "L_ON";
+    };
+
+
+    for( const auto &light_loc : here.find_furnitures_with_flag_in_omt( examp, flag ) ) {
+        g->m.furn_set( light_loc, g->m.get_furn_transforms_into( light_loc ) );
```

Comment:
```suggestion
        here.furn_set( light_loc, here.get_furn_transforms_into( light_loc ) );
```

the map instance is already obtained via `here`; avoid using the `g`ame god object.